Tests/OnlineRetailPortal.Tests/ProductMockTestClass.cs

```diff
@@ -0,0 +1,65 @@
+using OnlineRetailPortal.Contracts;
+using System;
+using Xunit;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OnlineRetailPortal.Mock;
+
+namespace OnlineRetailPortal.Tests
+{
+    public class ProductMockTestClass
```

Comment:
please go through the comments given by Nilesh on test cases and make the changes for all test classes (wherever applicable)

Reply:
done